src/main/resources/hudson/plugins/logparser/LogParserPublisher/config.jelly

```diff
@@ -9,7 +9,7 @@
   <f:entry title="Show log parser graphs" help="/plugin/log-parser/parser_graphs.html">
     <f:checkbox name="log-parser.showGraphs" checked="${instance.showGraphs}"/>
   </f:entry>
-  <f:radioBlock title="Use global rule" name="log-parser.useProjectRule" value="false" checked="${instance.useProjectRule!=true}">
+  <f:radioBlock title="Use global rule" name="log-parser.useProjectRule" value="false" checked="${instance.useProjectRule!=true}" inline="true">
```

Comment:
Did you check if there some acceptance test relying on this? The `inline` attribute changes the selector to access the field in acceptance tests.